docs/software/media.md

```diff
@@ -843,4 +846,105 @@ See also
+
+![DietPi media server software BruteFIR](../assets/images/dietpi-software-media-brutefir.gif){: style="width:500px"}
+
+=== "Configuration and Settings"
+
+    The configuration and settings are located there:
+
+    - `/etc/BruteFIR/null-65k48-l.pcm`: Null DRC filter (left channel)
+    - `/etc/BruteFIR/null-65k48-r.pcm`: Null DRC filter (right channel)
+    - `/etc/BruteFIR/highpass_l.pcm`: High pass DRC filter for testing (left channel)
+    - `/etc/BruteFIR/highpass_r.pcm`: High pass DRC filter for testing (right channel)
+    - `/etc/asound.conf`: Contains the BruteFIR ALSA loopback settings.
+    - `/var/log/brutefir`: Contains the BruteFIR log file.
+    - `/etc/BruteFIR/brutefir_config`: This file contains the EQ and DRC `.pcm` file location settings to be used.
+
+    If you make any changes, you will need to restart BruteFIR for the changes to take effect.
+
+=== "Restart BruteFIR service"
+
+    Restart the BruteFIR service with:
+
+    ```sh
+    dietpi-services restart
+    ```
+
+=== "HiFiBerry / External sound card"
+
+    If you are running a 24 bit external sound card, you can increase the sound quality by using `S32_LE` instead of `S16_LE`.
+
+    Change all `S16_LE` references to `S32_LE` in the following files:
+
+    - `/etc/asound.conf`
+    - `/etc/BruteFIR/brutefir_config`
+
+    Then restart BruteFIR.
+
+See also <https://torger.se/anders/brutefir.html>.
+
+## Raspotify - Spotify Connect client
+
+Spotify Connect client for the Raspberry Pi that Just Works™.
+
+!!! note "You will need a Spotify Premium account in order to use the Spotify Connect client."
+
+![DietPi media server software Raspotify](../assets/images/dietpi-software-media-raspotify.png){: style="width:250px"}
+
+Raspotify should work straight out of the box.
+
+=== "Changing options"
+
+    However, you can change the available options in:
+
+    - `/etc/default/raspotify`
+
+    Then restart services with:
+
+    ```sh
+    dietpi-services restart
+    ```
+
+See also: <https://github.com/dtcooper/raspotify>.
+
+## Spotify Connect Web - Web interface, client and player for Spotify Premium
+
+Spotify Connect Web is a console client and player for Spotify Connect including a web frontend.
+
+=== "Access to the web interface"
+
+    - URL = `http://<your.IP>:4000`
+
+=== "Generate the Spotify API key"
+
+    The generation of the `spotify_appkey.key` is required to allow the application, access to Spotify API using your account.
+
+    Fill out the application form <https://developer.spotify.com/documentation/general/guides/app-settings/#register-your-app>.  
+    Wait for it to be "accepted"
+
+=== "Transfer Spotify API key"
+
+    Make sure you have one of DietPi's [File Servers](https://dietpi.com/docs/software/file_servers/) installed.  
+    Copy the `spotify_appkey.key` to the following folder:
+
+    - folder = `/mnt/dietpi_userdata/spotify-connect-web`
+    - accessed from file server = `/spotify-connect-web`
+
```

Comment:
```
=== "View service logs"

    ```sh
    journalctl -u spotify-connect-web
    ```

=== "Service handling"

    ```
    systemctl start spotify-connect-web # Manually start the service
    systemctl stop spotify-connect-web # Manually stop the service
    systemctl restart spotify-connect-web # Manually restart the service
    ```
```

Reply:
Ok, added